src/index.html

```diff
@@ -4,9 +4,490 @@
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main
+      class="page__container catalog"
+    >
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <div class="card__container">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">
+            Product code: 195434
+          </div>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews-name">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+
+            <div class="card__price-amount">$2,199</div>
+          </div>
+
+          <a
+            href="#"
+            data-qa="card-hover"
+          >
+            <button
+              type="button"
+              class="card__button"
+            >
+              BUY
+            </button>
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <div class="card__container">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">
+            Product code: 195434
+          </div>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews-name">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+
+            <div class="card__price-amount">$2,199</div>
```

Comment:
Use `p` tag according semantic
```suggestion
            <p class="card__price-text">Price:</p>

            <p class="card__price-amount">$2,199</p>
```